include/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.hpp

```diff
@@ -227,26 +213,20 @@ class Manager
     Directory localRepository_;
     Duration localRepositoryLockTimeout_;
 
-    Array<BulletinA> aBulletins_;
-    Array<Finals2000A> finals2000aArray_;
+    BulletinA bulletinA_;
+    Finals2000A finals2000A_;
 
     mutable std::mutex mutex_;
 
-    mutable Index aBulletinIndex_;
-    mutable Index finals2000aIndex_;
-
-    mutable Instant bulletinAUpdateTimestamp_;
-    mutable Instant finals2000AUpdateTimestamp_;
-
     Manager(const Manager::Mode& aMode = Manager::DefaultMode());
 
     bool isLocalRepositoryLocked() const;
```

Comment:
Maybe all or none of the private mem funcs here should follow the ending with underscore_ naming pattern